lib/constants/index.ts

```diff
@@ -173,24 +192,14 @@ export const SANITY_STUDIO_API_PROJECT_ID = "dk34t4vc";
 export const SANITY_STUDIO_API_DATASET = "production";
 
 /** Tokens accepted as input for the offset aggregator /#/offset */
-export const inputTokens = [
-  "ubo",
-  "nbo",
-  "bct",
-  "nct",
-  "mco2",
-  "usdc",
-  "klima",
-  "sklima",
-  "wsklima",
-] as const;
-export type InputToken = typeof inputTokens[number];
+export const offsetInputTokens = allowancesContracts["retirementAggregator"];
+export type OffsetInputToken = typeof offsetInputTokens[number];
```

Comment:
We can do the assertion here, it's not really safer but at least it lets us remove the `any` from our component code here https://github.com/KlimaDAO/klimadao/pull/495/files#diff-b68c42fa04a583195c0f40a15e663dcebdd59c469a31bac00a34e1848e2a6c06R110

```suggestion
/** Tokens accepted as input for the offset aggregator /#/offset */
export type OffsetInputToken =
  typeof allowancesContracts["retirementAggregator"][number];
export const offsetInputTokens = allowancesContracts[
  "retirementAggregator"
] as unknown as OffsetInputToken[];
```

Reply:
Thanks! Nested assertions. Interesting. Typescript is a puzzle sometimes.